data/data/agent/files/usr/local/bin/common.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
```

Comment:
Add /usr/local/bin/common.sh to commonFiles() in https://github.com/openshift/installer/blob/fcaa07d1553c1b2cfed84642e7b38693c4af7ec9/pkg/asset/agent/image/ignition_test.go#L363 to fix the unit test issue.